admin-offline/src/main/java/ch/ge/ve/offlineadmin/controller/KeyGenerationController.java

```diff
@@ -74,6 +75,7 @@
     private StreamHasher streamHasher;
     private PasswordDialogController passwordDialogController;
     private KeyGenerator keyGenerator;
+    private ZoneId ChZoneId = ZoneId.of("Europe/Zurich");
```

Comment:
Same comment here. Maybe change the variable to `chZoneId` for naming consistency?